keep-ui/app/workflows/builder/editors.tsx

```diff
@@ -51,17 +51,17 @@ interface keepEditorProps {
   updateProperty: (key: string, value: any) => void;
   installedProviders?: Provider[] | null | undefined;
   providerType?: string;
+  type?: string;
```

Comment:
@KanvaBhatia where does this come from? who providers it?

Reply:
I used the `type` here - 
https://github.com/keephq/keep/blob/143c8eb48db5ec291e53d8981a4e1003344c663f/keep-ui/app/workflows/builder/editors.tsx#L398C2-L416C11

It is coming from here - 
https://github.com/keephq/keep/blob/143c8eb48db5ec291e53d8981a4e1003344c663f/keep-ui/app/workflows/builder/editors.tsx#L391C3-L392C21

It's coming from `useStepEditor()` of `sequential-workflow-designer-react`